src/locales/en.json

```diff
@@ -960,6 +960,7 @@
         "enableRatioColors": "Enable ratio colors",
         "fileContentInterval": "Torrent file content refresh interval",
         "hideChipIfUnset": "Hide chips if unset",
+        "hideColoredChip": "Hide colored chips (Text with color)",
```

Comment:
I would opt for something more explicit like
```suggestion
        "useTextForChips": "Use text variant for chips",
```

Reply:
> I would opt for something more explicit like

these format seems better. I will get this implemented